index.html

```diff
@@ -10,74 +10,112 @@
 </head>
 
 <body>
-  <header>
-    <h1>Jelena de la Sauce</h1>
-    <nav class="main-nav">
-      <a href="#contact" class="button">Contact</a>
+  <header class="container-head">
+    <h1 class="child-head-name">Jelena de la Sauce</h1>
+    <nav class="child-head-nav main-nav">
+      <a href="#contact"  class="button">Contact</a>
+      <a href="#about" class="button">About</a>
       <a href="#biography" class="button">Biography</a>
     </nav>
   </header>
 
-  <figure>
-    <img src="https://media.istockphoto.com/photos/view-from-above-of-home-office-desk-with-laptop-flowers-and-leaves-on-picture-id1030888824?b=1&k=20&m=1030888824&s=170667a&w=0&h=PdSVMCNoVesDaB53lOtkq8Vkjq7wgrSzRaKY0ElbL3Q=" alt="" width="400" height="250" class="coverimg centerimg"> 
-    <figcaption>I am a well organized and communicative Web Development student, who is graduating the boothcamp at neuefische end of March 2022. I am higly motivated to join a young, creative and inspiring team.</figcaption>
+  <summary class="container-sum">
+  <figure class="child-sumpic">
+    <div>
```

Comment:
das `figure`-Element verpackt das `img`-Tag wohl hinsichtlich Positionierung und Dokumentfluss recht gut - ich bin mir nicht sicher, inwieweit das `div`-Tag da zusätzlich drin good practice ist oder vielleicht sogar stört 
i am quite sure you have an `alt`attribute in the `img`tag but i cannot navigate to the end of this line of code 😅

Reply:
Yes, I also don't think the `div` is strictly necessary. And yes, we can't see the end of the line of code, unfortunately.

Reply:
Thanks both of you, I am still not 100% sure when to use it and when not 😅
And strange that you cannot see it - cant you scroll to the side? any idea if i should do sth different?